### PR TITLE
[Servicefabric] Migrate servicefabric to track2 SDK

### PR DESCRIPTION
**Pull request by PARADISSEEKR**
**Description**

Here are main changes for Backup migration to track2:

- Update live test(`test_managed_app_type` `test_managed_application` `test_managed_service` `test_managed_app_type_version`).
- Stateless service no longer has parameter `instance_close_delay_duration`.

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Servicefabric

**Comment by qwordy** [comment]
https://clitestresultstac.blob.core.windows.net/20210520-052743-844796/servicefabric.report.parallel.html
It looks good.

**Review by qwordy — APPROVED** [review]

File: src/azure-cli/azure/cli/command_modules/servicefabric/tests/latest/test_sf_managed_application.py [hunk 1]
```diff
@@ -235,12 +234,10 @@ def test_managed_service(self):
                          self.check('properties.replicaRestartWaitDuration', '00:11:00')])
         # Update Stateless Service
         self.cmd('az sf managed-service update -g {rg} -c {cluster_name} --application-name {app_name} --service-name {stateless_service_name} '
-                 '--min-instance-count {min_instance_count} --min-instance-percentage {min_instance_percentage} '
-                 '--instance-close-delay-duration {instance_close_delay_duration}',
+                 '--min-instance-count {min_instance_count} --min-instance-percentage {min_instance_percentage} ',
```
Comment (qwordy) [inline src/azure-cli/azure/cli/command_modules/servicefabric/tests/latest/test_sf_managed_application.py:237]: Should we remove it from command parameters?

Comment (PARADISSEEKR) [reply src/azure-cli/azure/cli/command_modules/servicefabric/tests/latest/test_sf_managed_application.py:237]: Model StatelessServiceProperties no longer has parameter instance_close_delay_duration.

**Pull request closed without merging**